MultiThreading.Task6.Continuation/Program.cs

```diff
@@ -24,7 +29,33 @@ static void Main(string[] args)
 
             // feel free to add your code
 
+            var firstTask = Task.Factory.StartNew(() => First());
+
+            var secondTask = firstTask.ContinueWith(
+                      prev => Second(),
+                      TaskContinuationOptions.AttachedToParent);
+
+            var onError = secondTask.ContinueWith(
+                                  prev => Console.WriteLine(prev.Exception),
+                                  TaskContinuationOptions.OnlyOnFaulted | TaskContinuationOptions.AttachedToParent);
+
+            var onSuccess = secondTask.ContinueWith(
+                      prev => Console.WriteLine("secondTask success"),
+                      TaskContinuationOptions.OnlyOnRanToCompletion | TaskContinuationOptions.AttachedToParent);
```

Comment:
when the parent task failed and parent task thread should be reused for continuation

Reply:
Have no idea how to do this? Could you please share some clue?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskcontinuationoptions?view=net-7.0

See an option to run on the same thread